CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 All notable changes to the z/OS FTP Plug-in for Zowe CLI will be documented in this file.
 
+## Recent Changes
+- Fix Windows path problem.
```

Comment:
Could the changelog update be more specific, to mention the problem was with the `delete uss-file` command? Thanks 🙂

Reply:
Thanks @t1m0thyj. I updated the changelog to make it clear.😊